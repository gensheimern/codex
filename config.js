module.exports = {
	DB_HOST: process.env.DB_HOST || '159.65.115.221',
	DB_USER: process.env.DB_USER || 'root',
	DB_PASS: process.env.DB_PASS || 'teamcodex',
	DB_NAME: process.env.DB_NAME || 'lunch_planner',
	JWT_SECRET: process.env.JWT_SECRET || 'secret',
	MAIL_HOST: process.env.MAIL_HOST || '159.65.115.221',
	MAIL_USER: process.env.MAIL_USER || 'support@codex-team.de',
	MAIL_PASS: process.env.MAIL_PASS || 'teamcodex123',
};
