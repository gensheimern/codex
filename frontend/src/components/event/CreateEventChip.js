import React from 'react';
import Avatar from 'material-ui/Avatar';
import Chip from 'material-ui/Chip';


const styles = {
  chip: {
    margin: 4,
  },
  wrapper: {
    display: 'flex',
    flexWrap: 'wrap',
  },
};

function handleRequestDelete() {
  alert('You clicked the delete button.');
}

function handleClick() {
  alert('You clicked the Chip.');
}

/**
 * Examples of Chips, using an image [Avatar](/#/components/font-icon), [Font Icon](/#/components/font-icon) Avatar,
 * [SVG Icon](/#/components/svg-icon) Avatar, "Letter" (string) Avatar, and with custom colors.
 *
 * Chips with the `onRequestDelete` property defined will display a delete icon.
 */
export default class ChipExampleSimple extends React.Component {

  handleDelete(){
    console.log("worked");
  }

  render() {
    return (
      <div style={styles.wrapper}>

        <Chip
          onRequestDelete={handleRequestDelete}
          onClick={handleClick}
          style={styles.chip}
        >
          <Avatar src={this.props.peopleImage} />
         {this.props.name}
        </Chip>

      </div>
    );
  }
}
