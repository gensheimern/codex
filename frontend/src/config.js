const config = {
	debug: process.env.REACT_APP_DEBUG === 'TRUE',
	apiPath: process.env.REACT_APP_API_PATH,
	wsPath: process.env.REACT_APP_WS_PATH,
	basePath: process.env.REACT_APP_BASE_PATH,
}

export default config;