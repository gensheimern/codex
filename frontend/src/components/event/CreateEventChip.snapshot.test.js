/**
 * @author:Stella Neser
 *  Komponente ist ein Snapshottest
 */
import React from 'react';
import renderer from'react-test-renderer';
import CreateEventChip from './CreateEventChip';
import config from '../../config';
import {MemoryRouter} from 'react-router-dom';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

const fetchMock = require('fetch-mock');

describe('CreateEventChip  Snapshot', () => {
    test('renders', () =>{
        const component = renderer.create(
            <MemoryRouter>
                <MuiThemeProvider>
                    <CreateEventChip />
                </MuiThemeProvider>
            </MemoryRouter>
        );
        let tree= component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});