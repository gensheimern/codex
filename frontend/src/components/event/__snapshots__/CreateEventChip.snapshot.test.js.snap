// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateEventChip  Snapshot renders 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexWrap": "wrap",
    }
  }
>
  <div
    className={undefined}
    disabled={undefined}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
        "backgroundColor": "rgb(224, 224, 224)",
        "border": 10,
        "borderRadius": 16,
        "boxShadow": null,
        "boxSizing": "border-box",
        "cursor": "pointer",
        "display": "flex",
        "fontFamily": "Roboto, sans-serif",
        "fontSize": "inherit",
        "fontWeight": "inherit",
        "margin": 4,
        "muiPrepared": true,
        "outline": "none",
        "padding": 0,
        "position": "relative",
        "textDecoration": "none",
        "verticalAlign": null,
        "whiteSpace": "nowrap",
        "width": "fit-content",
      }
    }
    tabIndex={0}
  >
    <div
      className={undefined}
      size={32}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgb(188, 188, 188)",
          "borderRadius": "50%",
          "color": "#ffffff",
          "display": "inline-flex",
          "fontSize": 16,
          "height": 32,
          "justifyContent": "center",
          "marginRight": -4,
          "muiPrepared": true,
          "userSelect": "none",
          "width": 32,
        }
      }
    />
    <span
      style={
        Object {
          "color": "rgba(0, 0, 0, 0.87)",
          "fontSize": 14,
          "fontWeight": 400,
          "lineHeight": "32px",
          "muiPrepared": true,
          "paddingLeft": 12,
          "paddingRight": 12,
          "userSelect": "none",
          "whiteSpace": "nowrap",
        }
      }
    />
  </div>
</div>
`;
